environments/common/src/main/java/org/jboss/weld/environment/deployment/discovery/jandex/Jandex.java

```diff
@@ -0,0 +1,40 @@
+package org.jboss.weld.environment.deployment.discovery.jandex;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Set;
+
+import org.jboss.weld.bootstrap.api.Bootstrap;
+import org.jboss.weld.environment.util.Reflections;
+import org.jboss.weld.resources.spi.ResourceLoader;
+
+/**
+ * @author Tomas Remes
+ */
+public class Jandex {
+
+    public static final String INDEX_ATTRIBUTE_NAME = JandexDiscoveryStrategy.class.getPackage().getName() + ".index";
+
+    private static final String JANDEX_INDEX_CLASS_NAME = "org.jboss.jandex.Index";
+    public static final String JANDEX_DISCOVERY_STRATEGY_CLASS_NAME = "org.jboss.weld.environment.deployment.discovery.jandex.JandexDiscoveryStrategy";
+    public static final String JANDEX_SERVLET_CONTEXT_BEAN_ARCHIVE_HANDLER = "org.jboss.weld.environment.servlet.deployment.JandexServletContextBeanArchiveHandler";
```

Comment:
I'm not so sure it makes sense to put a Weld Servlet specific class name here...